### ISSUE TITLE
question: Question about deps

### ISSUE THREAD

**Issue by fcater** [apitable/apitable#1667]
## When I run "pnpm i --ignore-scripts" in root directory, I see lots of deprecated packages, am I doing it right?

![image](https://github.com/apitable/apitable/assets/68488755/e41ecbc2-ed3b-467f-8ecd-632e87f0b572)

**Comment by mr-kelly** [comment]
Yes, this is expected. Like most large Node.js monorepos, some deep sub-dependencies might be marked as deprecated by their authors over time. As long as the build succeeds and there are no critical security vulnerabilities flagged by our CI, you can safely ignore these warnings.
